src/main/java/org/quiltmc/loader/impl/util/mappings/MixinIntermediaryDevRemapper.java

```diff
@@ -142,6 +142,11 @@ public String mapMethodName(String owner, String name, String desc) {
 			}
 		}
 
+		// Realistically, this can't be remapped.
+		if (owner.startsWith("java/")) {
+			return name;
+		}
+
```

Comment:
You could probably move this to the top of the method? Other than that this looks good.

Reply:
Think I placed it after the first `if` to have the HashSet get a hit in case it's already hashed by a previous call, but I could move it to the top as well.

Reply:
It does seem that the likely hood of it being a hit is practically none if it's directly from Mixin, so, readability wins here.

Reply:
![image](https://user-images.githubusercontent.com/8528256/213299425-9501fff5-0838-44eb-ad6c-a253454f1292.png)

Tho, it does seem to be less trivial given the fact that it can be null, unless I do an `owner != null` check before comparing strings.